posthog-android/src/main/java/com/posthog/android/replay/PostHogMaskModifier.kt

```diff
@@ -0,0 +1,20 @@
+package com.posthog.android.replay
+
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.SemanticsPropertyKey
+import androidx.compose.ui.semantics.semantics
+
+public object PostHogMaskModifier {
+    internal val PostHogReplayMask = SemanticsPropertyKey<Boolean>("ph-no-capture")
```

Comment:
we can make a const for this ("ph-no-capture") as well since it's used many times